src/Orleans.Runtime/Silo/Silo.cs

```diff
@@ -882,7 +874,7 @@ private void DomainUnobservedExceptionHandler(object context, UnhandledException
         {
             var exception = (Exception)args.ExceptionObject;
             if (context is ISchedulingContext)
-                UnobservedExceptionHandler(context as ISchedulingContext, exception);
+                UnobservedExceptionHandler(context as ISchedulingContext, exception, this.logger);
```

Comment:
Also have to migrate Silo off Logger because I want to make `UnObservedExceptionHandler` static.